### PR TITLE
policy: limit the number of L4 ports to 40

### PR DESCRIPTION
**Pull request by aalemayhu**
Closes: #922 (L4 policy with more than 40 ports doesn't work)

### CODE REVIEW

File: Documentation/policy.rst [hunk 1]
```diff
@@ -324,6 +324,9 @@ The ``toPorts`` field takes a ``PortProtocol`` structure which is defined as fol
                 Protocol string `json:"protocol,omitempty"`
         }
 
+.. note:: There is currently a max limit of 40 ports. This might change in the
+          future when support for ranges are added.
```
Comment (ianvernon) [inline Documentation/policy.rst]: is added

File: tests/11-getting-started.sh [hunk 2]
```diff
@@ -27,7 +26,7 @@ echo "------ checking cilium status ------"
 cilium status
 
 echo "------ creating Docker network of type Cilium ------"
```
Comment (ianvernon) [inline tests/11-getting-started.sh]: Remove the echo statements prior to the calls to this helper function as the echo statement is already part of the helper function.

File: Documentation/policy.rst (hunk 1 above)
Comment (aalemayhu) [reply Documentation/policy.rst]: Fixed

File: tests/11-getting-started.sh (hunk 2 above)
Comment (aalemayhu) [reply tests/11-getting-started.sh]: Was actually done earlier but probably pushed from the wrong machine. Should be gone now.

**Review by aanm — APPROVED** [review]

**Pull request merged**